driver/main_args.ml

```diff
@@ -409,7 +409,12 @@ let mk_ppx f =
 
 let mk_keywords f =
   "-keywords", Arg.String f,
-  "<version><list>   use the same keywords as <version> extended by <list>"
+  "<version+list>  set keywords following the <version+list> spec:\n
+  \                -<version> if present specifies the base set of keywords\n
+  \                  (if absent the current set of keywords is used)
+  \                 -<list> is a \"+\"-separated list of keywords to add to\n
```

Comment:
the spacing looks a bit odd here, I think the second list item is one space to the right of the first item